pharmd-app/src/themes/field-colors.js

```diff
@@ -78,15 +78,15 @@ const getPercentageColor = ({ record }) => {
 
 const getStudentStatusColor = ({ record }) => {
   switch (record[STUDENT.STATUS]) {
-    case STUDENT_STATUS.COOP:
+    case "Coop":
```

Comment:
Hmm I would prefer the use of the original enums here instead of a string. Maybe you can extract the code you wrote in ChipField into a method that you can call in ChipField and on the OG enums here? Idk if this is too extra lol.

Reply:
i too would like to preserve enums, i can take another stab at it and see what to do